run_bot.sh

```
@@ -52,9 +52,21 @@ check_docker() {
 
 # Build Docker image
 build_image() {
+    local image_name=${1:-meet-teams-bot}
+    local date_tag=$(date +%Y%m%d-%H%M)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)